docs/UserGuide.md

```diff
@@ -956,6 +956,201 @@ Important notes:
 > > <img src="images/unicash/command-outputs/addTransactionFailedOutput3.png" width="1000" />
 
 #### 4.1.2 EditTransactionCommand
+Edits an existing transaction in UniCa$h.
+
+Command: `edit_transaction INDEX [n/NAME] [type/TYPE] [amt/AMOUNT] [dt/DATETIME] [l/LOCATION] [c/CATEGORY]`
+
+Command Options:
+
+| Option Name | Optional? | Purpose                                                                                       |
```

Comment:
Maybe you can add a remark column here on the right, and specify the constraints

Reply:
Yeah, I think this is a good idea. I've implemented this fix and moved most of the constraints to this additional column to reduce the number of "Important notes"!